### ISSUE TITLE
Tests should run with all the existent process providers. 

### ISSUE THREAD

**Issue by sbragagnolo** [sbragagnolo/taskit#7]
Maybe it would be a good idea to use the parametrized tests from noury? In any case all the tests should run with all the running possibilities.

**Comment by guillep** [comment]
Maybe for a next release. I'd leave this issue open. Whaddayathink?

**Comment by sbragagnolo** [comment]
Yes indeed . I wrote it down to do not forget. next version. We have to discuss how to parametrize that .
